### PR TITLE
letter checker, logic

### PR DESCRIPTION
**Pull request by c-werdel**
Part three is the game play logic loop. This ticket will compare the users inputted letters to the letters in word_choice (word_choice selected word from website). It will determine if that letter inputted matches any of the letters in word_choice. If it is true than it will show the letter at its corresponding spot. If it is false it will tell the user that letter is wrong and put that wrong letter in a wrong guesses(list) and will show us that list.

### CODE REVIEW

**Comment by c-werdel** [comment]
Can you please review this new PR for me

File: hangman.py [hunk 1]
```diff
@@ -12,8 +12,32 @@ def get_word(myfile):
...
+        if guess not in word_choice:
+            print("Sorry that letter is not in the word")
+            wrong_guesses.append(guess)
+            print("Guessed letters: " , wrong_guesses)
+
 if __name__ == "__main__":
     myfile = read_words()
     choice_word = get_word(myfile)
-    print(choice_word)
+    play = letter_checker(choice_word)
```
Comment (dev-gilbride) [inline hangman.py]: Don't need the play var as letter_checker() doesn't have a return

File: hangman.py [hunk 2]
```diff
@@ -12,8 +12,32 @@ def get_word(myfile):
...
+            if letter == guess:
+                print("Correct that letter is in the word")
+                user_line[index_count] = guess
+            index_count += 1
+        print(user_line)
+
+        if guess not in word_choice:
+            print("Sorry that letter is not in the word")
+            wrong_guesses.append(guess)
+            print("Guessed letters: " , wrong_guesses)
```
Comment (dev-gilbride) [inline hangman.py]: You will want to break out the print statement to occur on correct and incorrect guesses, it's important context in either case.

**Review by dev-gilbride — CHANGES_REQUESTED** [review]
Added some comments. Namely, the wrong_guesses needs to be printed for correct and incorrect. Also it's called "Guessed letters" but doesn't include correct letters that have been guessed. Either call that list "Wrong guesses" or include correct and incorrect guessed letters in the list.

**Review by dev-gilbride — APPROVED** [review]
Looks good!

**Pull request merged**